code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -138,7 +138,7 @@
 
 /obj/item/reagent_containers/syringe/update_icon_state()
 	var/rounded_vol = get_rounded_vol()
-	icon_state = "[rounded_vol]"
+	icon_state = "[base_icon_state]_[rounded_vol]"
```

Comment:
base_icon_state need to be "syringe" on the regular syringes

Reply:
Done thanks for the help